ui/src/components/post-listing.tsx

```diff
@@ -253,6 +254,8 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
 
   listing() {
     let post = this.props.post;
+    const { creator_name, creator_published } = post;
```

Comment:
Just use `post.creator_published here`, none of the other props do this.

Reply:
Changed in 8fe8836bc263be11895fafa958409499c9e9fc9d.